pkgs/applications/misc/msglint/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
```

`rec` is missing here. Thus, the `version` variable is not available below.

```suggestion
stdenv.mkDerivation rec {
```